app/components/settings/project_custom_fields/header_component.rb

```diff
@@ -32,6 +32,10 @@ class HeaderComponent < ApplicationComponent
       include ApplicationHelper
       include OpPrimer::ComponentHelpers
       include OpTurbo::Streamable
+      def breadcrumbs_items
+        [{ href: admin_index_path, text: t("label_administration") },
```

Comment:
Clicking that link results in a crash of the page, as it violates the content secruity policy.

Reply:
Might be related to https://community.openproject.org/projects/openproject/work_packages/53813/activity

Reply:
Problem still exists